CONTRIBUTING.md

```diff
@@ -150,7 +150,7 @@ Once you have the additional numbers, you can setup additional storage profiles
 between them using the `NODE_APP_INSTANCE` environment variable.
 
 For example, to create an 'alice' profile, put a file called `local-alice.json` in the
-`config` directory:
+`config` directory, a child of the Signal directory, not `appData` from above:
 
```

Comment:
In my experience, the `appData` directory is the same as the `config` directory listed here. Have you seen differently? Maybe we could let go of that `config` term?

Reply:
On macOS, we have "~/Library/Application Support/" whereas `config` appears as something like `/Volumes/development/Signal/config.` So is `appData` the same as `config` on other platforms?

Reply:
Ah, you are right - I'm mistaken. Maybe it's best to call that something like 'the `/config` subdirectory of your project checkout?' We could potentially clarify that it should go alongside all the other `.json` config files there?

Reply:
Yes, that helps clarify the location.